packages/core-contracts/contracts/proxy/AbstractProxy.sol

```diff
@@ -1,9 +1,7 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "./ProxyStorage.sol";
-
-contract ForwardingProxy is ProxyStorage {
+abstract contract AbstractProxy {
```

Comment:
This contract is now only in charge of forwarding stuff. It does not specify where the implementation is retrieved from.

E.g. in UUPSProxy it gets it from local storage.
In BeaconProxy it gets it from the beacon contract.